public/keymaps/m/mini_elixivy_default.json

```
@@ -0,0 +1,22 @@
+{   
+    "keyboard": "mini_elixivy",
+    "keymap": "default_d997cad",
+    "commit": "d997cad04f3c253e6f04bf4848e6378f8c3af3db",
```

```suggestion
    "keymap": "default",
    "commit": "8392729d7410246ab3228d47df272da5b0c72c99",
```

1. The issue that disallowed `"keymap": "default"` is no longer an issue.
2. `8392729d7410246ab3228d47df272da5b0c72c99` because https://github.com/qmk/qmk_firmware/commit/8392729d7410246ab3228d47df272da5b0c72c99